.buildkite/pipeline.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Generate Buildkite pipelines dynamically"""
+
+import json
+
+INSTANCES = [
+    "m5d.metal",
+    "m6i.metal",
+    "m6a.metal",
+    "m6gd.metal",
+]
+
+KERNELS = ["4.14", "5.10"]
```

Comment:
You can use here the SUPPORTED_KERNELS variable from  framework.defs.

Reply:
I think it may be too much trouble for what's it's worth. Would have to set up `PYTHONPATH` etc, and we may want to use the constants for different purposes.